app/src/main/java/com/goldenraven/padawanwallet/ui/settings/SettingsRootScreen.kt

```diff
@@ -33,159 +37,167 @@ internal fun SettingsRootScreen(
+                    .padding(start = 24.dp, end = 24.dp, top = 24.dp)
+            ) {
+                Text(
+                    text = "About",
+                    fontWeight = FontWeight.Normal,
+                    color = Color(0xff2f2f2f)
 
-        Row(
-            horizontalArrangement = Arrangement.Center,
-            modifier = Modifier.fillMaxWidth()
-        ) {
-            Text(
-                text = "Padawan Wallet ${BuildConfig.VERSION_NAME}",
-                style = PadawanTypography.bodySmall,
-                color = padawan_theme_text_faded_secondary
+                )
+                Spacer(modifier = Modifier.weight(1f))
+                Icon(
+                    painter = painterResource(id = R.drawable.ic_hicon_right_2),
+                    contentDescription = "Scan icon",
+                    tint = Color(0xff2f2f2f)
+                )
+            }
+
+            Divider(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 40.dp, bottom = 8.dp, start = 40.dp, end = 40.dp),
+                color = Color(0xff8f8f8f),
+                thickness = 1.dp
             )
-        }
 
-        Divider(
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(top = 8.dp, bottom = 40.dp, start = 40.dp, end = 40.dp),
-            color = Color(0xff8f8f8f),
-            thickness = 1.dp
-        )
+            Row(
+                horizontalArrangement = Arrangement.Center,
+                modifier = Modifier.fillMaxWidth()
+            ) {
+                Text(
+                    text = "Padawan Wallet ${BuildConfig.VERSION_NAME}",
+                    style = PadawanTypography.bodySmall,
+                    color = padawan_theme_text_faded_secondary
+                )
+            }
 
-        Button(
-            onClick = {
-                viewModel.unsetAllCompleted()
-            },
-            colors = ButtonDefaults.buttonColors(containerColor = Color(0xffFEE6DE)),
-            shape = RoundedCornerShape(8.dp),
-            border = BorderStroke(1.dp, padawan_theme_button_primary),
-            modifier = Modifier
-                .size(width = 400.dp, height = 60.dp)
-                .padding(start = 24.dp, end = 24.dp)
-        ) {
-            Text(
-                text = "Reset completed chapters",
-                fontWeight = FontWeight.Normal,
-                color = Color(0xff2f2f2f)
+            Divider(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 8.dp, bottom = 40.dp, start = 40.dp, end = 40.dp),
+                color = Color(0xff8f8f8f),
+                thickness = 1.dp
             )
+
+            Button(
+                onClick = {
+                    viewModel.unsetAllCompleted()
+                    scope.launch { scaffoldState.snackbarHostState.showSnackbar("Chapters Reset Successful") }
```

Comment:
I tend to capitalize only the first letter in these small micro-copy items. Here that would be `Chapters reset successful`.

Reply:
Updated it 👍